jest.config.js

```diff
@@ -6,6 +6,9 @@ const foremanReactRelative = 'webpack/assets/javascripts/react_app';
 const foremanFull = foremanLocation();
 const foremanReactFull = foremanRelativePath(foremanReactRelative);
 
+// Makes graphql files work in test
+tfmConfig.transform["\\.(gql|graphql)$"] = "jest-transform-graphql";
```

Comment:
Do we need to add this library to `@theforeman/test`? It looks like it hasn't been updated in 4 years though - https://www.npmjs.com/package/jest-transform-graphql